webpack/assets/javascripts/foreman_hosts.js

```diff
@@ -0,0 +1,26 @@
+export let pluginEditAttributes = {
```

Comment:
I would prefer keeping the list of attributes internal to this module and expose a method that allows registering more attributes, something like `tfm.hosts.registerAttributes('os', ['lifecycle_environment_id', 'content_view_id', 'environment_id', 'content_source_id', 'architecture_id'])` for example.